clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -3513,6 +3533,7 @@ public void testEmptyGroupId(GroupProtocol groupProtocol) {
         assertEquals("The configured group.id should not be an empty string or whitespace.", e.getCause().getMessage());
     }
 
+    @SuppressWarnings("resource")
```

Comment:
It looks like this is redundant.

Reply:
IntelliJ warning. newConsumer returns an auto-closable